<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage_Election</class>
 <widget class="QMainWindow" name="Manage_Election">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Starts At</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="start_date">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="date">
     <date>
      <year>2016</year>
      <month>11</month>
      <day>5</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ends At</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="start_time">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="time">
     <time>
      <hour>20</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QTimeEdit" name="end_time">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="time">
     <time>
      <hour>21</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QDateEdit" name="end_date">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="date">
     <date>
      <year>2016</year>
      <month>11</month>
      <day>5</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="abort">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src/files.qrc">
      <normaloff>:/abort-png-hi.png</normaloff>:/abort-png-hi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="result">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src/files.qrc">
      <normaloff>:/button_generate-result.png</normaloff>:/button_generate-result.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src/files.qrc">
      <normaloff>:/back-button.png</normaloff>:/back-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stats">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src/files.qrc">
      <normaloff>:/button_view-election-stats.png</normaloff>:/button_view-election-stats.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>175</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src/files.qrc">
      <normaloff>:/button_save-changes.png</normaloff>:/button_save-changes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>150</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="src/files.qrc"/>
 </resources>
 <connections/>
</ui>
